source/administration/cloud_billing.rtf

```diff
@@ -0,0 +1,90 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1671\cocoasubrtf600
+{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\margl1440\margr1440\vieww10800\viewh8400\viewkind0
+\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 =========================\
+Mattermost Cloud Billing\
```

Comment:
I think this may be a duplicate of this content: https://docs.mattermost.com/overview/mattermost-cloud-overview.html. Happy to move it as needed but just want to make sure it should be in the admin guide and not the cloud admin guide (and not the currently-published content).

Reply:
@justinegeffen Yes, that is correct.  Proposing replacing the content in cloud overview with the generalized content on Professional and Enterprise and then updating the cloud admin billing page with the billing content.